examples/tabs/tabs-manual.html

```diff
@@ -43,23 +46,74 @@ <h2 id="ex_label">Example</h2>
+              <p><a href="https://en.wikipedia.org/wiki/Maria_Theresia_Ahlefeldt">Maria Theresia Ahlefeldt</a> (16 January 1755 – 20 December 1810) was a Danish, (originally German), composer. She is known as the first female composer in Denmark. Maria Theresia composed music for several ballets, operas, and plays of the royal theatre. She was given good critic as a composer and described as a “<span lang="da">virkelig Tonekunstnerinde</span>” ('a True Artist of Music').</p>
             </div>
-
-            <div tabindex="0" role="tabpanel" id="agnes-tab" aria-labelledby="agnes" class="is-hidden">
-              <p>Agnes Caroline Thaarup Obel is a Danish singer/songwriter. Her first album, Philharmonics, was released by PIAS Recordings on 4 October 2010 in Europe. Philharmonics was certified gold in June 2011 by the Belgian Entertainment Association (BEA) for sales of 10,000 Copies.</p>
+            <div
+              id="tabpanel-2"
+              role="tabpanel"
+              aria-labelledby="tab-2"
+              class="is-hidden">
+              <p><a href="https://en.wikipedia.org/wiki/Joachim_Andersen_(composer)">Carl Joachim Andersen</a> (29 April 1847 – 7 May 1909) was a Danish flutist, conductor and composer born in Copenhagen, son of the flutist Christian Joachim Andersen. Both as a virtuoso and as composer of flute music, he is considered one of the best of his time. He was considered to be a tough leader and teacher and demanded as such a lot from his orchestras but through that style he reached a high level.</p>
             </div>
-
-            <div tabindex="0" role="tabpanel" id="complex-complex" aria-labelledby="complex" class="is-hidden">
-              <p>Fear of complicated buildings:</p>
-              <p>A complex complex complex.</p>
+            <div
+              id="tabpanel-3"
+              role="tabpanel"
+              aria-labelledby="tab-3"
+              class="is-hidden">
+              <p><a href="https://en.wikipedia.org/wiki/Ida_Henriette_da_Fonseca">Ida Henriette da Fonseca</a> (July 27, 1802 – July 6, 1858) was a Danish opera singer and composer.  Ida Henriette da Fonseca was the daughter of Abraham da Fonseca (1776–1849) and Marie Sofie Kiærskou (1784–1863). She and her sister Emilie da Fonseca were students of Giuseppe Siboni, choir master of the Opera in Copenhagen. She was given a place at the royal Opera alongside her sister the same year she debuted in 1827.</p>
+            </div>
+            <div
+              id="tabpanel-4"
+              role="tabpanel"
+              aria-labelledby="tab-4"
+              class="is-hidden">
+              <p><a href="https://en.wikipedia.org/wiki/Peter_Lange-M%C3%BCller">Peter Erasmus Lange-Müller</a> (1 December 1850 – 26 February 1926) was a Danish composer and pianist. His compositional style was influenced by Danish folk music and by the work of Robert Schumann; Johannes Brahms; and his Danish countrymen, including J.P.E. Hartmann.</p>
```

Comment:
Two main issues:
1.  I am not sure it is a a good practice for the APG to reference contemporary musicians, it might be misunderstood as some type of endorsement for those musicians or at least an unfair improvement to their SEO, so I did not feel comfortable using them in the updates I made. 
2. We will be adding at least one more example that will add and delete tabs and this was easier for me to find additional related content for those examples.